main.rb

```diff
@@ -0,0 +1,122 @@
+module Enumerable
+  def my_each
+    return unless block_given?
+
+    each do |i|
+      yield i
+    end
+    self
+  end
```

Comment:
# my_each

- [ ] `my_each` returns an Enumerator if no block is given
     Failure/Error: expect(array.my_each).to be_an(Enumerator)

       expected nil to be a kind of Enumerator